src/test/java/uk/gov/pay/publicauth/filters/LoggingFilterTest.java

```diff
@@ -81,7 +81,7 @@ public void shouldLogEntryAndExitPointsOfEndPoints() throws Exception {
         String endLogMessage = loggingEvents.get(1).getFormattedMessage();
         assertThat(endLogMessage, containsString(format("[%s] - %s to %s ended - total time ", requestId, requestMethod, requestUrl)));
         String[] timeTaken = StringUtils.substringsBetween(endLogMessage, "total time ", "ms");
-        assertTrue(NumberUtils.isNumber(timeTaken[0]));
+        assertTrue(NumberUtils.isCreatable(timeTaken[0]));
```

Comment:
This replaces the use of the deprecated `NumberUtils.isNumber(String)` method with the replacement suggested by its Javadoc